OpenSSL/SSL.py

```diff
@@ -1551,6 +1551,16 @@ def get_cipher_version(self):
             return version.decode("utf-8")
 
 
+    def get_protocol_version(self):
+        """
+        Obtain the protocol version of the current connection.
+
+        :returns: The TLS version of the current connection, for example
+            the value for TLS 1.2 would be 0x303.
```

Comment:
It may be hard for most people to generalize from "0x303 means TLS 1.2" to an understanding of other values.  Or maybe that's just mean.  Anyway, can you expand this documentation or make it easier to understand these values some other way?  An example of a non-documentation solution might be to make the result self-documenting by defining a collection of symbolic constants and referring to that collection here.  Or another solution might be to refer to some existing OpenSSL documentation about this value.

Which brings me to another point, where is the documentation for `SSL_version`?  I can't find any.  I did find `SSL_get_version` which apparently returns a **string** instead.  Exposing that instead might be another way to solve the documentation issue - since "TLS1.2" doesn't need as much explanation as 0x303 (though just going by the OpenSSL documentation, I don't know if "TLSv1.2" is a value that will ever be returned by `SSL_get_version`!  It only documents "SSLv2", "SSLv3", and "TLSv1" - oh, and, awesomely, "unknown").